src/styles/index.scss

```diff
@@ -1,3 +1,104 @@
+@import "stars";
+
 body {
+  font-family: $family-roboto;
+  color: #060b35;
   margin: 0;
 }
+
+.card {
```

Comment:
The stylization of `card` classes could be transferred to a separate `card.scss` file, and in the `index.scss file` leave the styles for the `body` and transfer the `*`-styles here.